drivers/Makefile.features

```diff
@@ -0,0 +1,39 @@
+_802154_DRIVERS = at86rf2xx     \
+                  cc110x        \
+                  cc2420        \
+                  cc2538_rf     \
+                  kw2xrf        \
+                  mrf24j40      \
+                  xbee          \
+                  #
+
+_ETHERNET_DRIVERS = enc28j60    \
+                    encx24j600  \
+                    ethos       \
+                    w5100       \
+                    #
+
+_LORA_DRIVERS = rn2xx3          \
+                sx127x          \
+                #
+
+_NETIF_DRIVERS = $(_802154_DRIVERS)     \
+                 $(_ETHERNET_DRIVERS)   \
+                 sx127x                 \
+                 #
```

Comment:
Ping?

Reply:
I agree with @miri64, also I don't think we should extend on this global or section specific Makefiles, but rather put driver/module specific stuff in their respective Makefiles. Hence, the `sx127x` driver should simply define

```
FEATURES_PROVIDED += radio_lora
FEATURES_PROVIDED += netif
```

same for the rest. Looking at the discussion on lists like `BOARD_INSUFFICIENT_MEMORY` or black and white lists which we want to overcome, this would basically introduce driver lists that need to be maintained.

Reply:
@smlng, please read the full discussion, especially https://github.com/RIOT-OS/RIOT/pull/11676#issuecomment-537534671.